### DISCUSSION TITLE
Remote-Control natively from iOS?

### DISCUSSION THREAD

**Discussion by HerrDings** [mikebrady/shairport-sync#1299] [Q&A] [answered]
First of all, big thanks for this absolutely great and easy-to-use software! It really seems that you know what you are doing with sound and linux! 

I set up a shairport-sync on ubuntu running on a raspi4 (with [HiFiBerry Digi+](https://www.hifiberry.com/shop/boards/hifiberry-digiplus-standard-version/) and a PoE-Hat). This should replace my old AirPortExpress (2nd gen) and it worked from the spot.

My AirPort Express has a feature, that seems to be missing in Shairport Sync: Its ability be to remote-controlled by other i-Devices.

So in my example I played _Highway to Hell_ from my iPhone on my AirPort Express. Then I took my iPad and went to the control center. At the bottom there is a button to control other devices (see screenshot 1)
![img1](https://user-images.githubusercontent.com/37446122/134984202-d89d2361-148a-49ab-9254-522df868f3c4.jpg)

If I press it, the UI offers me a list with my Airport Express in it. (screenshot 2)

![img2](https://user-images.githubusercontent.com/37446122/134984351-2d482d7b-cf56-4f83-adeb-269fe3ba7f0d.jpg)

And if I tap it, I am able to control the music that is played on my iPhone.

![img3](https://user-images.githubusercontent.com/37446122/134984383-26eb264f-96ef-425d-9b44-c5a94f902336.jpg)

This feature seems to be missing in Shairport Sync. So my question is:

**Is there a switch in the config-file, I might have missed or is this really not supported?**

If it is not supported, please consider this a feature request.

**Comment by mikebrady** [comment, marked as answer]
Thanks for the post. Yes, we have come across this. Unfortunately, the protocols have not been published, and we have to figure them out by guesswork. We have a lot to figure out still!

**Reply by HerrDings** [reply]
Thanks for looking into this and for your answer.
Keep up the good work!
